package.json

```diff
@@ -13,10 +13,10 @@
     "run-elm": "lib/index.js"
   },
   "scripts": {
-    "lint": "eslint index.js",
+    "lint": "eslint src test --ignore-pattern elm-stuff",
     "pretest": "npm run build",
-    "test": "node tests/run-tests.js",
-    "build": "babel index.js --out-dir lib",
+    "test": "jest \\.test.js",
```

Comment:
I don't think we need the `\\.test.js` part here.

Reply:
You're right – [default value of `testMatch`](https://facebook.github.io/jest/docs/en/configuration.html#testmatch-array-string) suits us.